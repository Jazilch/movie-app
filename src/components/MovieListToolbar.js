'use es6';

import React from 'react';
import styled from 'styled-components/macro';
import MovieSearchContainer from '../containers/MovieSearchContainer';
import MovieFilterContainer from '../containers/MovieFilterContainer';

const MovieListToolbarWrapper = styled.div`
  display: flex;
  justify-content: space-between;
  align-items: center;
  height: 80px;
  padding: 0 1rem;
  margin-bottom: 1rem;
  background-color: #494949;
`;

const MovieListToolbar = () => {
  return (
    <MovieListToolbarWrapper>
      <MovieFilterContainer />
      <MovieSearchContainer />
    </MovieListToolbarWrapper>
  );
};

export default MovieListToolbar;
