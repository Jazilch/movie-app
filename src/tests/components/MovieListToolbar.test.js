/* eslint-disable no-undef */

'use es6';

import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import MovieListToolbar from '../../components/MovieListToolbar';

describe('MovieListToolbar Component', () => {
  it('renders the component correctly', () => {
    const movielisttoolbar = shallow(<MovieListToolbar />);
    expect(toJson(movielisttoolbar)).toMatchSnapshot();
  });
});
