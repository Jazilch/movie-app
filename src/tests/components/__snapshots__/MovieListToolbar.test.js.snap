// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieListToolbar Component renders the component correctly 1`] = `
<MovieListToolbar__MovieListToolbarWrapper>
  <Connect(MovieGenreFilter) />
  <Connect(MovieSearch) />
</MovieListToolbar__MovieListToolbarWrapper>
`;
